pkg/cloud/google/machineactuator.go

```diff
@@ -28,21 +28,19 @@ import (
 	"strings"
 	"time"
 
-	"github.com/golang/glog"
+	"github.com/ghodss/yaml"
```

Comment:
FYI I think we have `sigs.k8s.io/yaml` that is our preferred fork...

Reply:
+1